.github/workflows/model.yml

```diff
@@ -113,18 +113,6 @@ jobs:
       - name: Running transformers ulysses tests on 8 L20 GPUs + latest transformers
         run: |
           torchrun --nproc_per_node=8 -m pytest tests/models/test_transformers_ulysses.py
-      - name: Running transformers ulysses tests on 8 L20 GPUs + transformers 4.54.1
```

Comment:
We probably want to keep at least one pinned version in CI

Reply:
keep `transformers==4.54.1` for ci